### ISSUE TITLE
Pre-work Checklist: Developer: Adrian Ang

### ISSUE THREAD

**Issue by adrianang** [hackforla/website#4718]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [ ] Note: There are no meetings on the 1st-7th of every month.
- [ ] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [ ] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [ ] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [ ] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [ ] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [ ] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [ ] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by adrianang** [comment]
_How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?_

About 5 hours (+/- 30 mins!)
- ~2.5 hours reading documentation/project boards/setting up developer environment
- ~2.5 hours attending meetings

**Comment by adrianang** [comment]
### Progress Report — 7/7/2023

**Progress:** I got issue #4911 merged this week; have not yet reassigned myself to an issue but will do so over the weekend! 
**Blockers:** No current blockers.
**Availability:** 
 - Saturdays/Sundays: Generally from 10 AM - 10 PM PST
 - Weekdays, excluding Wednesdays: Generally from 6 PM - 10 PM PST
ETA: N/A

**Comment by adrianang** [comment]
### Progress Report — 7/28/2023

**Progress:** Still working on #2306; have mostly been reading through the resources on GHA and looking at PRs related to GHA/automation. 
**Blockers:** No current blockers.
**Availability:** 
 - Saturdays/Sundays: Generally from 2 PM - 8 PM PST
 - Weekdays, excluding Wednesdays: Generally from 6 PM - 10 PM PST

**ETA:** N/A

**Comment by ExperimentsInHonesty** [comment]
Hi @adrianang I was reviewing the work you have done on the team, and I wanted to encourage you to finish out the checklist by making an issue for other developers to work on.  Those issues can be found in the [ERs that are ready to be turned into issues column](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22#column-19761756).  I have filtered it by front end for you.

- [x] Setting up your local environment from a contributing file
- [x] GitHub branching
- [x] Pull Requests
- [x] Reviewed other people's Pull Requests
- [x] Resolved Merge Conflicts
- [x] Written documentation for other Developers (Architecture, etc.)
- [x] Mentored other developers
- [ ] Making issues for other developers to work on

**Comment by HackforLABot** [comment]
## Activity Log: adrianang
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- adrianang opened: https://github.com/hackforla/website/issues/4718 at 05/23/2023, 08:14 PM PDT
- adrianang assigned: https://github.com/hackforla/website/issues/4718 at 05/23/2023, 08:14 PM PDT
- adrianang assigned: https://github.com/hackforla/website/issues/4841 at 06/18/2023, 06:32 PM PDT
- adrianang unassigned: https://github.com/hackforla/website/issues/4841 at 06/18/2023, 06:39 PM PDT
- adrianang assigned: https://github.com/hackforla/website/issues/4790 at 06/18/2023, 06:46 PM PDT
- adrianang commented on: https://github.com/hackforla/website/issues/4790#issuecomment-1597423288 at 06/19/2023, 08:56 AM PDT
- adrianang commented on: https://github.com/hackforla/website/issues/4718#issuecomment-1597430022 at 06/19/2023, 09:00 AM PDT
- adrianang commented on: https://github.com/hackforla/website/issues/4790#issuecomment-1603951213 at 06/23/2023, 01:51 AM PDT
- adrianang issue closed as completed: https://github.com/hackforla/website/issues/4790#event-9619896536 at 06/23/2023, 10:09 AM PDT
- adrianang assigned: https://github.com/hackforla/website/issues/4529 at 06/23/2023, 11:17 AM PDT
- adrianang commented on: https://github.com/hackforla/website/issues/4529#issuecomment-1604753427 at 06/23/2023, 12:11 PM PDT
- adrianang opened: https://github.com/hackforla/website/pull/4882 at 06/25/2023, 10:41 PM PDT
- adrianang PR was merged: https://github.com/hackforla/website/pull/4882#event-9646988394 at 06/26/2023, 10:13 PM PDT
- adrianang issue closed by PR 4882: https://github.com/hackforla/website/issues/4529#event-9646988550 at 06/26/2023, 10:13 PM PDT
- adrianang assigned: https://github.com/hackforla/website/issues/2288#issuecomment-1586250052 at 06/26/2023, 10:47 PM PDT
- adrianang commented on: https://github.com/hackforla/website/issues/2288#issuecomment-1609029424 at 06/27/2023, 01:23 AM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/4890#issuecomment-1610955087 at 06/28/2023, 01:03 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/4890#pullrequestreview-1506721997 at 06/30/2023, 01:08 AM PDT
- adrianang commented on: https://github.com/hackforla/website/issues/2288#issuecomment-1616504956 at 07/02/2023, 02:13 AM PDT
- adrianang opened: https://github.com/hackforla/website/pull/4911 at 07/04/2023, 05:11 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/4896#pullrequestreview-1513542749 at 07/04/2023, 06:44 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/4896#pullrequestreview-1513688283 at 07/04/2023, 10:33 PM PDT
- adrianang PR was merged: https://github.com/hackforla/website/pull/4911#event-9735853515 at 07/05/2023, 12:55 PM PDT
- adrianang issue closed by PR 4911: https://github.com/hackforla/website/issues/2288#event-9735853720 at 07/05/2023, 12:55 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/4896#pullrequestreview-1518207502 at 07/07/2023, 12:50 AM PDT
- adrianang commented on: https://github.com/hackforla/website/issues/4718#issuecomment-1624992650 at 07/07/2023, 01:14 AM PDT
- adrianang assigned: https://github.com/hackforla/website/issues/4507 at 07/07/2023, 01:48 AM PDT
- adrianang commented on: https://github.com/hackforla/website/issues/4507#issuecomment-1625148299 at 07/07/2023, 02:44 AM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/4896#issuecomment-1627540829 at 07/08/2023, 04:15 PM PDT
- adrianang opened: https://github.com/hackforla/website/pull/4926 at 07/09/2023, 02:14 AM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/4896#issuecomment-1628386893 at 07/10/2023, 12:24 AM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/4925#issuecomment-1628475438 at 07/10/2023, 01:20 AM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/4925#issuecomment-1628506366 at 07/10/2023, 01:39 AM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/4928#issuecomment-1630103314 at 07/10/2023, 09:38 PM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/4896#issuecomment-1630110611 at 07/10/2023, 09:48 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/4928#pullrequestreview-1523669818 at 07/11/2023, 12:56 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/4896#pullrequestreview-1525639502 at 07/11/2023, 11:52 PM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/4928#issuecomment-1636716912 at 07/15/2023, 02:11 AM PDT
- adrianang PR was merged: https://github.com/hackforla/website/pull/4926#event-9830276834 at 07/15/2023, 08:06 PM PDT
- adrianang issue closed by PR 4926: https://github.com/hackforla/website/issues/4507#event-9830276847 at 07/15/2023, 08:06 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/4928#pullrequestreview-1531707387 at 07/16/2023, 02:09 AM PDT
- adrianang assigned: https://github.com/hackforla/website/issues/2306#issuecomment-1554142329 at 07/16/2023, 04:09 AM PDT
- adrianang commented on: https://github.com/hackforla/website/issues/2306#issuecomment-1637055358 at 07/16/2023, 04:12 AM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/4950#issuecomment-1637465475 at 07/16/2023, 11:42 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/4925#pullrequestreview-1533439791 at 07/17/2023, 11:35 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/4950#pullrequestreview-1538938476 at 07/20/2023, 03:21 AM PDT
- adrianang merged: https://github.com/hackforla/website/pull/4950#event-9872854633 at 07/20/2023, 03:42 AM PDT
- adrianang closed by PR 4950: https://github.com/hackforla/website/issues/3732#event-9872854848 at 07/20/2023, 03:42 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5006#pullrequestreview-1542079769 at 07/22/2023, 03:36 AM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5006#event-9890911305 at 07/22/2023, 03:37 AM PDT
- adrianang closed by PR 5006: https://github.com/hackforla/website/issues/4954#event-9890911332 at 07/22/2023, 03:37 AM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5043#issuecomment-1646559597 at 07/22/2023, 04:11 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5043#pullrequestreview-1542084036 at 07/22/2023, 04:30 AM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5044#issuecomment-1646563617 at 07/22/2023, 04:38 AM PDT
- adrianang commented on: https://github.com/hackforla/website/issues/4919#issuecomment-1646566548 at 07/22/2023, 04:57 AM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5046#issuecomment-1646568104 at 07/22/2023, 05:05 AM PDT
- adrianang closed: https://github.com/hackforla/website/pull/5046#event-9891016238 at 07/22/2023, 05:06 AM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5047#issuecomment-1646570722 at 07/22/2023, 05:18 AM PDT
- adrianang closed: https://github.com/hackforla/website/pull/5047#event-9891035205 at 07/22/2023, 05:19 AM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5047#issuecomment-1646657935 at 07/22/2023, 12:42 PM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5050#issuecomment-1646894135 at 07/23/2023, 10:10 AM PDT
- adrianang closed: https://github.com/hackforla/website/pull/5050#event-9893137292 at 07/23/2023, 10:10 AM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5051#issuecomment-1650835444 at 07/25/2023, 06:39 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5043#pullrequestreview-1548934839 at 07/26/2023, 08:19 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5043#event-9930866161 at 07/26/2023, 08:21 PM PDT
- adrianang closed by PR 5043: https://github.com/hackforla/website/issues/5027#event-9930866277 at 07/26/2023, 08:21 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5051#pullrequestreview-1548998641 at 07/26/2023, 09:55 PM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5037#issuecomment-1653389377 at 07/27/2023, 04:00 AM PDT
- adrianang commented on: https://github.com/hackforla/website/issues/4718#issuecomment-1655522780 at 07/28/2023, 04:25 AM PDT
- adrianang commented on: https://github.com/hackforla/website/issues/2306#issuecomment-1655524921 at 07/28/2023, 04:27 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5049#pullrequestreview-1551981675 at 07/28/2023, 04:55 AM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5049#event-9947301220 at 07/28/2023, 04:55 AM PDT
- adrianang closed by PR 5049: https://github.com/hackforla/website/issues/4415#event-9947301425 at 07/28/2023, 04:55 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5056#pullrequestreview-1551993112 at 07/28/2023, 05:04 AM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5056#event-9947377107 at 07/28/2023, 05:05 AM PDT
- adrianang closed by PR 5056: https://github.com/hackforla/website/issues/4939#event-9947377316 at 07/28/2023, 05:05 AM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5058#issuecomment-1655591755 at 07/28/2023, 05:20 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5058#pullrequestreview-1553206639 at 07/28/2023, 10:21 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5061#pullrequestreview-1553214803 at 07/28/2023, 11:04 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5061#event-9953848376 at 07/28/2023, 11:04 PM PDT
- adrianang closed by PR 5061: https://github.com/hackforla/website/issues/4809#event-9953848402 at 07/28/2023, 11:04 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5063#pullrequestreview-1553264162 at 07/29/2023, 03:42 AM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5063#event-9954270791 at 07/29/2023, 03:43 AM PDT
- adrianang closed by PR 5063: https://github.com/hackforla/website/issues/4813#event-9954270812 at 07/29/2023, 03:43 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5093#pullrequestreview-1553265458 at 07/29/2023, 04:01 AM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5093#event-9954289118 at 07/29/2023, 04:01 AM PDT
- adrianang closed by PR 5093: https://github.com/hackforla/website/issues/4798#event-9954289132 at 07/29/2023, 04:01 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5097#pullrequestreview-1553266367 at 07/29/2023, 04:12 AM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5097#event-9954302237 at 07/29/2023, 04:12 AM PDT
- adrianang closed by PR 5097: https://github.com/hackforla/website/issues/4802#event-9954302255 at 07/29/2023, 04:12 AM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5103#issuecomment-1656709785 at 07/29/2023, 04:30 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5107#pullrequestreview-1553268590 at 07/29/2023, 04:39 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5051#pullrequestreview-1553537594 at 07/29/2023, 07:21 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5058#pullrequestreview-1553538512 at 07/29/2023, 07:41 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5058#event-9955918991 at 07/29/2023, 07:43 PM PDT
- adrianang closed by PR 5058: https://github.com/hackforla/website/issues/5038#event-9955919021 at 07/29/2023, 07:43 PM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5095#issuecomment-1657003717 at 07/29/2023, 08:04 PM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5055#issuecomment-1657040122 at 07/29/2023, 09:32 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5055#pullrequestreview-1553578780 at 07/29/2023, 10:10 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5095#pullrequestreview-1553738935 at 07/30/2023, 05:21 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5095#event-9957754558 at 07/30/2023, 05:22 PM PDT
- adrianang closed by PR 5095: https://github.com/hackforla/website/issues/4917#event-9957754663 at 07/30/2023, 05:22 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5037#pullrequestreview-1553850783 at 07/30/2023, 07:49 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5037#event-9958515313 at 07/30/2023, 07:49 PM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5107#issuecomment-1657430221 at 07/30/2023, 08:04 PM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5107#issuecomment-1661926213 at 08/02/2023, 03:05 AM PDT
- adrianang closed: https://github.com/hackforla/website/pull/5107#event-9986188916 at 08/02/2023, 03:05 AM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5055#issuecomment-1661947666 at 08/02/2023, 03:20 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5112#pullrequestreview-1563794463 at 08/04/2023, 11:42 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5112#event-10014600205 at 08/04/2023, 11:44 PM PDT
- adrianang closed by PR 5112: https://github.com/hackforla/website/issues/5003#event-10014600284 at 08/04/2023, 11:44 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5122#pullrequestreview-1564021040 at 08/05/2023, 02:47 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5122#event-10016137336 at 08/05/2023, 02:47 PM PDT
- adrianang closed by PR 5122: https://github.com/hackforla/website/issues/4804#event-10016137352 at 08/05/2023, 02:47 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5128#pullrequestreview-1564024479 at 08/05/2023, 03:21 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5128#event-10016174387 at 08/05/2023, 03:21 PM PDT
- adrianang closed by PR 5128: https://github.com/hackforla/website/issues/4913#event-10016174398 at 08/05/2023, 03:21 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5130#pullrequestreview-1564042398 at 08/05/2023, 06:20 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5130#pullrequestreview-1564042457 at 08/05/2023, 06:21 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5130#event-10016360909 at 08/05/2023, 06:21 PM PDT
- adrianang closed by PR 5130: https://github.com/hackforla/website/issues/4299#event-10016360927 at 08/05/2023, 06:21 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5133#pullrequestreview-1564043992 at 08/05/2023, 06:37 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5133#event-10016377436 at 08/05/2023, 06:37 PM PDT
- adrianang closed by PR 5133: https://github.com/hackforla/website/issues/4805#event-10016377452 at 08/05/2023, 06:37 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5142#pullrequestreview-1564044877 at 08/05/2023, 06:49 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5142#event-10016391298 at 08/05/2023, 06:50 PM PDT
- adrianang closed by PR 5142: https://github.com/hackforla/website/issues/5139#event-10016391340 at 08/05/2023, 06:50 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5144#pullrequestreview-1564048152 at 08/05/2023, 07:27 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5145#pullrequestreview-1564049049 at 08/05/2023, 07:36 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5145#event-10016436820 at 08/05/2023, 07:36 PM PDT
- adrianang closed by PR 5145: https://github.com/hackforla/website/issues/5138#event-10016436832 at 08/05/2023, 07:36 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5123#pullrequestreview-1564051900 at 08/05/2023, 07:58 PM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5055#issuecomment-1666698231 at 08/05/2023, 08:38 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5144#event-10017148601 at 08/06/2023, 04:38 AM PDT
- adrianang closed by PR 5144: https://github.com/hackforla/website/issues/4808#event-10017148705 at 08/06/2023, 04:38 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5123#pullrequestreview-1568960214 at 08/09/2023, 12:52 AM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5123#event-10043592855 at 08/09/2023, 12:52 AM PDT
- adrianang closed by PR 5123: https://github.com/hackforla/website/issues/4801#event-10043593072 at 08/09/2023, 12:52 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5055#pullrequestreview-1568978189 at 08/09/2023, 01:02 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5055#pullrequestreview-1571476989 at 08/10/2023, 02:41 AM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5055#event-10057111324 at 08/10/2023, 02:43 AM PDT
- adrianang closed by PR 5055: https://github.com/hackforla/website/issues/4504#event-10057111736 at 08/10/2023, 02:43 AM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5235#issuecomment-1685127905 at 08/19/2023, 04:04 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5235#event-10137159601 at 08/19/2023, 04:04 PM PDT
- adrianang closed by PR 5235: https://github.com/hackforla/website/issues/5031#event-10137159611 at 08/19/2023, 04:04 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5240#pullrequestreview-1585767407 at 08/19/2023, 04:48 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5240#event-10137191538 at 08/19/2023, 04:48 PM PDT
- adrianang closed by PR 5240: https://github.com/hackforla/website/issues/5201#event-10137191549 at 08/19/2023, 04:48 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5241#pullrequestreview-1585767842 at 08/19/2023, 04:58 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5241#event-10137198065 at 08/19/2023, 04:59 PM PDT
- adrianang closed by PR 5241: https://github.com/hackforla/website/issues/5197#event-10137198073 at 08/19/2023, 04:59 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5243#pullrequestreview-1585768359 at 08/19/2023, 05:06 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5243#event-10137207237 at 08/19/2023, 05:07 PM PDT
- adrianang closed by PR 5243: https://github.com/hackforla/website/issues/5147#event-10137207260 at 08/19/2023, 05:07 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5245#pullrequestreview-1585775354 at 08/19/2023, 07:33 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5247#pullrequestreview-1585776358 at 08/19/2023, 07:58 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5247#event-10137355016 at 08/19/2023, 07:58 PM PDT
- adrianang closed by PR 5247: https://github.com/hackforla/website/issues/5150#event-10137355023 at 08/19/2023, 07:58 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5250#pullrequestreview-1585777471 at 08/19/2023, 08:23 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5245#pullrequestreview-1588245099 at 08/21/2023, 09:18 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5245#event-10152774355 at 08/21/2023, 09:19 PM PDT
- adrianang closed by PR 5245: https://github.com/hackforla/website/issues/5186#event-10152774426 at 08/21/2023, 09:19 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5263#pullrequestreview-1597063686 at 08/26/2023, 07:17 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5263#event-10201003635 at 08/26/2023, 07:18 PM PDT
- adrianang closed by PR 5263: https://github.com/hackforla/website/issues/5184#event-10201003687 at 08/26/2023, 07:18 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5276#pullrequestreview-1597064296 at 08/26/2023, 07:29 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5276#event-10201014279 at 08/26/2023, 07:29 PM PDT
- adrianang closed by PR 5276: https://github.com/hackforla/website/issues/5267#event-10201014294 at 08/26/2023, 07:29 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5283#pullrequestreview-1597064865 at 08/26/2023, 07:40 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5283#event-10201023588 at 08/26/2023, 07:41 PM PDT
- adrianang closed by PR 5283: https://github.com/hackforla/website/issues/5199#event-10201023611 at 08/26/2023, 07:41 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5284#pullrequestreview-1597065755 at 08/26/2023, 07:57 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5284#event-10201036433 at 08/26/2023, 07:57 PM PDT
- adrianang closed by PR 5284: https://github.com/hackforla/website/issues/5256#event-10201036442 at 08/26/2023, 07:57 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5262#pullrequestreview-1597066262 at 08/26/2023, 08:07 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5262#event-10201045904 at 08/26/2023, 08:08 PM PDT
- adrianang closed by PR 5262: https://github.com/hackforla/website/issues/5185#event-10201045922 at 08/26/2023, 08:08 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5294#pullrequestreview-1597067585 at 08/26/2023, 08:32 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5294#event-10201065578 at 08/26/2023, 08:33 PM PDT
- adrianang closed by PR 5294: https://github.com/hackforla/website/issues/5173#event-10201065596 at 08/26/2023, 08:33 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5295#pullrequestreview-1597068313 at 08/26/2023, 08:47 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5295#event-10201076166 at 08/26/2023, 08:48 PM PDT
- adrianang closed by PR 5295: https://github.com/hackforla/website/issues/5140#event-10201076178 at 08/26/2023, 08:48 PM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5296#issuecomment-1694562699 at 08/26/2023, 08:57 PM PDT
- adrianang commented on: https://github.com/hackforla/website/issues/2306#issuecomment-1694639204 at 08/27/2023, 04:13 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5397#pullrequestreview-1608393726 at 09/02/2023, 05:00 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5397#event-10265025949 at 09/02/2023, 05:00 PM PDT
- adrianang closed by PR 5397: https://github.com/hackforla/website/issues/5369#event-10265025978 at 09/02/2023, 05:00 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5398#pullrequestreview-1608394787 at 09/02/2023, 05:14 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5399#pullrequestreview-1608395485 at 09/02/2023, 05:25 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5399#event-10265059623 at 09/02/2023, 05:26 PM PDT
- adrianang closed by PR 5399: https://github.com/hackforla/website/issues/5153#event-10265059643 at 09/02/2023, 05:26 PM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5400#issuecomment-1703973538 at 09/02/2023, 05:35 PM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5398#pullrequestreview-1609254118 at 09/04/2023, 03:34 AM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5398#event-10272381662 at 09/04/2023, 03:34 AM PDT
- adrianang closed by PR 5398: https://github.com/hackforla/website/issues/4466#event-10272381851 at 09/04/2023, 03:34 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5643#pullrequestreview-1663189975 at 10/08/2023, 01:50 AM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5643#event-10583601226 at 10/08/2023, 01:51 AM PDT
- adrianang closed by PR 5643: https://github.com/hackforla/website/issues/5177#event-10583601265 at 10/08/2023, 01:51 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5659#pullrequestreview-1663193343 at 10/08/2023, 02:15 AM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5659#event-10583640378 at 10/08/2023, 02:15 AM PDT
- adrianang closed by PR 5659: https://github.com/hackforla/website/issues/5631#event-10583640417 at 10/08/2023, 02:15 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5663#pullrequestreview-1663198975 at 10/08/2023, 02:39 AM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5663#event-10583687013 at 10/08/2023, 02:41 AM PDT
- adrianang closed by PR 5663: https://github.com/hackforla/website/issues/5628#event-10583687044 at 10/08/2023, 02:41 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5664#pullrequestreview-1663202578 at 10/08/2023, 03:04 AM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5664#event-10583722898 at 10/08/2023, 03:05 AM PDT
- adrianang closed by PR 5664: https://github.com/hackforla/website/issues/5319#event-10583722928 at 10/08/2023, 03:05 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5658#pullrequestreview-1663205403 at 10/08/2023, 03:23 AM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5667#issuecomment-1751993138 at 10/08/2023, 03:36 AM PDT
- adrianang closed: https://github.com/hackforla/website/pull/5667#event-10583793974 at 10/08/2023, 03:36 AM PDT
- adrianang submitted review: https://github.com/hackforla/website/pull/5658#pullrequestreview-1669593507 at 10/10/2023, 06:35 PM PDT
- adrianang merged: https://github.com/hackforla/website/pull/5658#event-10611988446 at 10/10/2023, 06:35 PM PDT
- adrianang closed by PR 5658: https://github.com/hackforla/website/issues/5588#event-10611988566 at 10/10/2023, 06:35 PM PDT
- adrianang commented on: https://github.com/hackforla/website/issues/5747#issuecomment-1774063999 at 10/22/2023, 04:07 AM PDT
- adrianang closed as completed: https://github.com/hackforla/website/issues/5747#event-10733078149 at 10/22/2023, 04:07 AM PDT
- adrianang reopened: https://github.com/hackforla/website/issues/5747#event-10733078149 at 10/22/2023, 04:08 AM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5748#issuecomment-1774064441 at 10/22/2023, 04:10 AM PDT
- adrianang closed: https://github.com/hackforla/website/pull/5748#event-10733081742 at 10/22/2023, 04:10 AM PDT
- adrianang commented on: https://github.com/hackforla/website/issues/5747#issuecomment-1774064796 at 10/22/2023, 04:11 AM PDT
- adrianang closed as not planned: https://github.com/hackforla/website/issues/5747#event-10733084482 at 10/22/2023, 04:11 AM PDT
- adrianang opened: https://github.com/hackforla/website/pull/5834 at 11/03/2023, 02:02 AM PDT
- adrianang commented on: https://github.com/hackforla/website/pull/5834#issuecomment-1793710886 at 11/05/2023, 03:32 AM PST
- adrianang PR was merged: https://github.com/hackforla/website/pull/5834#event-10906130262 at 11/08/2023, 01:43 PM PST
